src/Microsoft.IdentityModel.Logging/LogHelper.cs

```diff
@@ -446,7 +446,7 @@ private static LogEntry WriteEntry(EventLevel level, Exception innerException, s
                 _isHeaderWritten = true;
 
                 LogEntry headerEntry = new LogEntry();
-                headerEntry.EventLevel = EventLevel.LogAlways;
+                headerEntry.EventLogLevel = EventLogLevel.Informational;
```

Comment:
@jennyf19, I know we would like to see the header information at all log levels. Unfortunately, there is no equivalent to EventLevel.LogAlways in AspNetCore's LogLevel enum. Set the EventLogLevel to Informational as that is our recommended LogLevel. Thoughts on how this can be improved?

Reply:
Can it be not tied to a log level? Would it always be written in that case? That can also be a separate PR.

Reply:
Each message has to be tied to a log level. If we were to set it to Critical it might be color coded in red (not sure) and customers may not like it so settled for Information.

Reply:
Where are we doing the mapping between EventLevel/EventLogLevel and LogLevel ([https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loglevel](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loglevel))?

Reply:
I think I get it a bit more than this morning in the meeting, seems like we should just collapse critcal and LogAlways in the mapping to LogLevel, I think the worry about displaying red shouldn't be too concerning, LogAlways is already something that says, "People should always see this" right?

Reply:
> Where are we doing the mapping between EventLevel/EventLogLevel and LogLevel (https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loglevel)?

Mapping from EventLevel/EventLogLevel to LogLevel is done in the Identity Libraries/SDKs that implement IIdentityLogger.